flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_UpperECAM.js

```diff
@@ -1,3 +1,5 @@
+// TazX - 06/08/2021: Engine imbalance has been hardcoded in the Wasm. All references have been eliminated.
```

Comment:
You don't need to leave comments on what you did, `git blame` can do that

Reply:
Deleted comments on both Upper and Lower ECAM js files.